src/xdp/trn_kern.h

```diff
@@ -254,8 +254,8 @@ static inline void trn_update_l4_csum(__u64 *csum, __be32 old_addr,
 }
 
 __ALWAYS_INLINE__
-static inline void trn_update_l4_csum_port(__u64 *csum, __be16 old_port,
-					   __be16 new_port)
+static inline void trn_update_l4_csum_port(__u64 *csum, __be32 old_port,
+					   __be32 new_port)
```

Comment:
ports are 16 bit numbers, why do we need to change it to 32 bit?

Reply:
I'll check this in for now as I don't see any obvious regressions and based on your testing that verified it fixes the instability for service connectivity. But I think we need to rewrite the checksum code. It is not trivial (except for the fix I made earlier) and I don't want to hold up the release for it.